baseapp/baseapp.go

```diff
@@ -505,17 +447,17 @@ func (app *BaseApp) runTx(mode runTxMode, txBytes []byte, tx sdk.Tx) (result sdk
 
 	// Run the ante handler.
 	if app.anteHandler != nil {
-		newCtx, result, abort := app.anteHandler(ctx, tx)
+		var newCtx sdk.Context
```

Comment:
should the ante handler really get an empty context? why not the `ctx` we already populated above?

Reply:
Tried to modify it, and the test failed :( I don't know why does it affect to the execution but we cannot change it.